app/assets/javascripts/discourse/tests/index.html

```diff
@@ -26,6 +26,14 @@
         -o-transition: none !important;
         transition: none !important;
       }
+      #qunit-fixture {
```

Comment:
Ember CLI makes this visible for some reason which is quite jarring when it happens in browser.